Missionframework/modules/02_core/fnc/fn_core_intro.sqf

```diff
@@ -21,31 +21,33 @@
 --- TODO IN SPRINT .... soon ^^ ---
 Rework the intro and cinematic. Currently it doesn't work reliable. Mostly an endless loop of the cinematic.
 Postponed, as it's not in the current scope of the sprint 5 tasks.
+
+Also: Refactor or even remove it completely
```

Comment:
Any reason to keep dead code? Same concern as config, we can check old code via git.

Reply:
I wanted to have a discussion in the next days in Discord concerning keeping the intro cinematic or not. Until we come to a decision I haven't really cared much about this particular code/function.